custom/champ/tests/__init__.py

```diff
@@ -60,5 +64,13 @@ def tearDownModule():
         'corehq.apps.callcenter.data_source.call_center_data_source_configuration_provider'
     )
     _call_center_domain_mock.start()
+    with override_settings(SERVER_ENVIRONMENT='production'):
```

Comment:
why is this override required? may be a small comment would help?

Reply:
I'm not sure this is necessary actually. I'll test

Reply:
b0f51f4149d